### PR TITLE
Got moved again.

### PR DESCRIPTION
**Pull request by ralphbean**

### CODE REVIEW

File: bugzilla2fedmsg.py [hunk 1]
```diff
@@ -84,8 +84,8 @@ def convert_datetimes(obj):
 
 class BugzillaConsumer(moksha.hub.api.Consumer):
 
-    # This is /topic/com.redhat.bugzilla in STOMP land.
-    topic = '/topic/com.redhat.bugzilla'
+    # This is the fedora_from_esb A-MQ queue.
```
Comment (pypingou) [inline bugzilla2fedmsg.py]: from esb?

**Comment by pypingou** [comment]
Code wise :+1:

File: bugzilla2fedmsg.py (hunk 1 above)
Comment (puiterwijk) [inline bugzilla2fedmsg.py]: I guess "esb" = "Enterprise Service Bus"

Comment (ralphbean) [inline bugzilla2fedmsg.py]: More like "egregious spaghetti box" :wink:

Comment (pypingou) [inline bugzilla2fedmsg.py]: I'm in!

**Comment by ralphbean** [comment]
Thanks guys!  :)

**Pull request merged**